### PR TITLE
goto 1.2.3 (new formula)

### PR DESCRIPTION
**Pull request by iridakos**
This formula provides the [goto](https://github.com/iridakos/goto) shell script.

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
*The repo is younger than 30 days*
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by iridakos** [comment]
The problem is:
```
* GitHub repository too new (<30 days old)
```

Should I close this PR and come back after the required audit period or let it here as is?

**Comment by vinyldarkscratch** [comment]
Hey @iridakos, looking at the repo, it doesn't meet the watchers and forks count requirements either.  I'd recommend creating your own tap to host this formula?

**Comment by iridakos** [comment]
Hello @vinyldarkscratch ,

Based on [this](https://github.com/Homebrew/brew/blob/b4443e1cc0b41d2b25ca15b87c3d4cb2ab65837f/Library/Homebrew/dev-cmd/audit.rb#L485) I think it meets the requirement (it has more stars than the required):

```ruby
if formula&.tap&.core_tap? &&
       (metadata["forks_count"] < 20) && (metadata["subscribers_count"] < 20) &&
       (metadata["stargazers_count"] < 50)
      problem "GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)"
end
```

**Comment by vinyldarkscratch** [comment]
Hey @iridakos, yes, it has more than enough stars.  However, if you notice the `&&` instead of `||`, it must meet *all* of the requirements for stars, watchers, and forks.  You're so close, though!

**Comment by iridakos** [comment]
@vinyldarkscratch I do notice the `&&`: if those conditions are **all true** then that would be a `problem` which are not so there should not be a problem :)

**Comment by vinyldarkscratch** [comment]
Oh, I'm so sorry, I read that entirely backwards.  My bad!  That's actually surprising to me, I've seen some repositories turned down because they didn't have enough forks, but they had enough stars.  I guess they changed it.  😛

So yeah, you're right, then -- once the repo is 30 days old, you should be able to get the formula added!

**Comment by iridakos** [comment]
No worries! Waiting it is then... :watch: :eyes:

**Comment by vigo** [comment]
this 30 days thing is really annoying... i'd like to start using it right now... please, rethink about it

**Comment by bsormagec** [comment]
yes , i agree to @vigo , we want to use it please merge it ASAP.

File: Formula/goto.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Goto < Formula
+  desc "Shell tool for navigation to aliased directories with auto-completion"
+  homepage "https://github.com/iridakos/goto"
+  url "https://github.com/iridakos/goto/archive/v1.2.2.tar.gz"
+  sha256 "611ef7b82f6798ef83db0473811740c7f1ff5a15bb4ff52a1e5841bac87ba7f5"
+
+  bottle :unneeded
+
+  def install
+    bash_completion.install "goto.sh" => "goto"
```
Comment (fxcoudert) [inline Formula/goto.rb]: Is the goto script specific to bash?

Comment (fxcoudert) [inline Formula/goto.rb]: If the name of the script in the package is `goto.sh` then we should keep that, not rename it. Principle of least surprise.

File: Formula/goto.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  test do
+    assert_match "-F _complete_goto_bash",
+                 shell_output("source #{bash_completion}/goto && complete -p goto")
```
Comment (fxcoudert) [inline Formula/goto.rb]:
```
    output = shell_output("source #{bash_completion}/goto && complete -p goto")
    assert_match "-F _complete_goto_bash", output
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/goto.rb (hunk 1 above)
Comment (iridakos) [reply Formula/goto.rb]: Yes, at the moment it has only been tested in Bash under macOS.

Comment (iridakos) [reply Formula/goto.rb]: Correct, brewing fix.

File: Formula/goto.rb (hunk 2 above)
Comment (iridakos) [reply Formula/goto.rb]: Brewing fix.

File: Formula/goto.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/goto.rb]: Then let's edit the description to be `Bash tool` instead of `Shell tool`

Comment (iridakos) [reply Formula/goto.rb]: Sure thing, brewing fix.

Comment (iridakos) [reply Formula/goto.rb]: Question: there has been a new release. May I change the url as well pointing it to it? If yes, I will also update this thread to "goto 1.2.2 (new formula)"

Comment (fxcoudert) [reply Formula/goto.rb]: Yes, update everything. 1 commit.

Comment (iridakos) [reply Formula/goto.rb]: > Yes, update everything. 1 commit.

Sorry, I already sent the previous fixes explicitly for each review note...

Comment (iridakos) [reply Formula/goto.rb]: Done.

**Comment by fxcoudert** [comment]
Thanks @iridakos for the pull request!

**Comment by iridakos** [comment]
@fxcoudert Thank you very much for your work on this!

@vigo @bsormagec Thank you very much for your interest in this formula! I hope it works and you like it :D

**Comment by bsormagec** [comment]
@iridakos Thanks from us ;) with @vigo, now we can use it 👍

**Comment by iridakos** [comment]
@bsormagec Of course neighbor! :+1:  :+1:  :+1:

**Comment by vigo** [comment]
yeah! great! this tool made me remember good old AMIGA OS's `ASSIGN` directive. Almost same... Thanx @iridakos great work man!

**Comment by iridakos** [comment]
You're welcome @vigo, very glad to hear that!

**Pull request merged**